parsely-amp-tracking.php

```diff
@@ -0,0 +1,25 @@
+add_action( 'pre_amp_render_post', 'parsely_amp_add_actions' );
+function parsely_add_amp_actions() {
+    add_filter( 'amp_post_template_analytics', 'parsely_amp_add_analytics' );
+}
+
+function parsely_amp_add_analytics( $analytics ) {
+    global $parsely;
+    $parsely_options = $parsely->get_options();
```

Comment:
The plugin calls this object `$parselyOptions`; we'll set it here with `$parselyOptions = $this->get_options();`